.eslintrc.js

```diff
@@ -28,5 +28,33 @@ module.exports = {
     "max-statements-per-line": "off",
     "new-cap":                 "off",
     "operator-linebreak":      "off"
-  }
-};
\ No newline at end of file
+  },
+  overrides: [
+    // node files
+    {
+      files: [
+        'ember-cli-build.js',
+        'index.js',
+        'testem.js',
+        'blueprints/*/index.js',
+        'config/**/*.js',
+        'tests/dummy/config/**/*.js'
+      ],
+      excludedFiles: [
+        'addon/**',
+        'addon-test-support/**',
+        'app/**',
+        'tests/dummy/app/**'
+      ],
```

Comment:
What does this mean? Exclude files from eslint?

Reply:
I guess so, these are changes that were introduced by the ember update